rust/azure_iot_operations_connector/src/deployment_artifacts/azure_device_registry.rs

```diff
@@ -432,20 +432,11 @@ impl TryFrom<String> for DeviceEndpointRef {
 
     fn try_from(value: String) -> Result<Self, Self::Error> {
         // The below assumes the format is always {device_name}_{inbound_endpoint_name} with no additional
-        // `_` in the names.
-        // Kubernetes does not allow `_` in the names so this should be safe.
+        // `_` in the device name. The inbound endpoint name may have additional `_`.
```

Comment:
```suggestion
        // `_` in the device name. The inbound endpoint name may have additional `_`.
        // Kubernetes does not allow `_` in the device name so this should be safe.
```
seems still worth calling out